README.md

```diff
@@ -123,7 +123,7 @@ be developed for this milestone.
       and Firewood.
 - [ ] :runner: Support replicating the full state with corresponding range proofs that
       verify the correctness of the data.
-- [ ] Pluggable IO subsystem (tokio\_uring, monoio, etc)
+- [ ] Pluggable IO subsystem (tokio_uring, monoio, etc)
```

Comment:
Unfortunately, an underscore is a special character in markdown, so not having the escape means the markdown renderer has to look for another underscore later in the file. Different implementations may behave differently as to how far forward it will search for the closing underscore.

As of right now, I don't think there is another underscore anywhere else in the file.

Why was this backslash removed?

Reply:
+1

Reply:
Whoops, that was my formatter. I think an underscore surrounded by letters cannot be used for italics which is why my formatter does that. 

I didn't mean to add this change though, I'll remove it.